addons/web/static/tests/legacy/views/list_tests.js

```diff
@@ -4535,7 +4537,7 @@ QUnit.module('Views', {
             arch: '<tree><field name="foo"/></tree>',
             viewOptions: {
                 action: {
-                    help: '<p class="hello">click to add a record</p>'
+                    help: markup('<p class="hello">click to add a record</p>'),
```

Comment:
There's half a dozen of these, plus a dozen more in other view tests, but aside from a pair they seem unnecessary to markup or entirely unnecessary: most of the tests don't even check for them, and the rest generally just verifies that the "no content helper" is included by looking for `.o_view_nocontent`.

Very few tests bother checking for the actual content, let alone legitimately do so. But that creates quite a lot of noise in the security warnings.

Reply:
It will generate the noise once, so I don't think it's a big deal. Also, those markup occurrences in the test files could be ignored by the security check.

Reply:
> It will generate the noise once

Yeah but the noise is unnecessary since these are mostly not used at all.

Reply:
I removed them